src/content-handlers/iiif/JQueryPlugins.ts

```diff
@@ -441,11 +441,17 @@ export default function jqueryPlugins($) {
     lines: number,
     lessText: string,
     moreText: string,
+    lessAriaLabelPrefix: string,
+    moreAriaLabelPrefix: string,

```

Comment:
It might be better to add these new arguments to the end of the list and give them default values so there's less fragility if somebody is calling the helper using the old signature.

Reply:
Good point! I've added the English strings as the defaults for now.